actions.yml

```
@@ -10,6 +10,9 @@ inputs:
   branch:
     description: Branch where changes should be pushed too
     required: true
+  file_pattern:
+    description: File pattern used for "git add"
+    default: '.'
```

```suggestion
    required: false
    default: '.'
```

The `required` property is [mandatory](https://help.github.com/en/github/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#inputsinput_idrequired).